### PR TITLE
Flex layout extensions doc

### PR DESCRIPTION
**Pull request by danielftz**

### CODE REVIEW

**Comment by PRMerger6** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/flex-layout-extensions.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+The `Basis` extension method allows you to set the amount of space that's allocated to a `View` in `FlexLayout` on the main axis. The size can be specified in device-independent units, as a percentage of the size of the `FlexLayout` or based on the `View`'s requested width or height. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#basis).
+
+## Grow
+
+The `Grow` extension method specifies the amount of available spacea `View` in `FlexLayout` should use on the main axis. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#grow).
+
+## Order
+
+The `Order` extension method allows you to change the order that the children of the FlexLayout are arranged.  Setting this property overrides the order that it appears in the `Children` collection. For further detail refer to the[Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#order).
```
Comment (jfversluis) [inline docs/maui/markup/extensions/flex-layout-extensions.md:30]:
```suggestion
The `Order` extension method allows you to change the order that the children of the FlexLayout are arranged.  Setting this property overrides the order that it appears in the `Children` collection. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#order).
```

File: docs/maui/markup/extensions/flex-layout-extensions.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Shrink
+
+The `Shrink` extension method allows you to indicate which `View` in `FlexLayout` is given priority in being displayed at their full sizes when the aggregate size of `Children` is greater than 
+on the main axis. For further detail refer to the[Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#shrink).
```
Comment (jfversluis) [inline docs/maui/markup/extensions/flex-layout-extensions.md:35]:
```suggestion
on the main axis. For further detail refer to the [Microsoft documentation](/dotnet/maui/user-interface/layouts/flexlayout#shrink).
```

File: docs/maui/markup/extensions/flex-layout-extensions.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+```csharp
+using Microsoft.Maui.Layouts;
+using CommunityToolkit.Maui.Markup;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+
+namespace CommunityToolkit.Maui.Markup.Sample.Pages;
+
+public class FlexLayoutSamplePage : ContentPage
```
Comment (jfversluis) [inline docs/maui/markup/extensions/flex-layout-extensions.md:48]: Would it make sense to divide up the sample code a bit more and put it under the specific headings like we do for [other Markup docs](https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/extensions/bindable-object-extensions)?

@brminnick opinions?

**Review by jfversluis — CHANGES_REQUESTED** [review]
Thank you so much for providing this functionality and documentation! A couple of minor things that we should be able to resolve pretty fast :)

In addition to the other comments, make sure to add an entry in the TOC file for this!

**Comment by PRMerger7** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/flex-layout-extensions.md (hunk 3 above)
Comment (danielftz) [reply docs/maui/markup/extensions/flex-layout-extensions.md:48]: > In addition to the other comments, make sure to add an entry in the TOC file for this!

I'm not sure where in TOC file I would add it

Comment (jfversluis) [reply docs/maui/markup/extensions/flex-layout-extensions.md:48]: See this PR as an example: https://github.com/MicrosoftDocs/CommunityToolkit/pull/50/files

**Comment by PRMerger12** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Pull request merged**